client/app/components/onboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -29,18 +30,25 @@ const GuideCoverPhotoPage = (props) => {
       }),
       div({
         className: css.infoTextContent,
-        dangerouslySetInnerHTML: { __html: t('advice') }, // eslint-disable-line react/no-danger
-      }),
+      }, [
+        t('web.admin.onboarding.guide.cover_photo.advice.content1',
+          { link: a({
+            href: 'http://support.sharetribe.com/knowledgebase/articles/744438',
+            target: '_blank',
```

Comment:
Maybe not relevant in this case (although I'd still change it to be sure), but I'll remind about this every time I see a link with `target="_blank"`:

https://mathiasbynens.github.io/rel-noopener/